norminette/errors.py

```diff
@@ -172,9 +176,20 @@ def __init_subclass__(cls) -> None:
 
 class HumanizedErrorsFormatter(_formatter):
     def __str__(self) -> str:
+
+        def colorize(status: Literal["OK", "Error", "Notice"], txt: str) -> str:
+            return "\033[9" + (
+                "1" if status == "Error" else (
+                    "2" if status == "OK" else "3"
+                )) + "m" + txt + "\033[0m"
+
         output = ''
         for file in self.files:
-            output += f"{file.basename}: {file.errors.status}!"
+            status = file.errors.status
+            if status == "OK" and file.errors.has_notice:
+                status = "Notice"
+            output += colorize(status, f"[{'KO' if status == 'Error' else 'OK'}]")
+            output += f" {file.basename}"
```

Comment:
What about creating `colorful_status` property in `Errors` to include this changes? Because `has_notice` will be used only here, the same with `colorize`.

Reply:
Changed it!